includes/frontend.php

```diff
@@ -216,6 +216,9 @@ public function init() {
 
 		// Add Edit with the Elementor in Admin Bar.
 		add_action( 'admin_bar_menu', [ $this, 'add_menu_in_admin_bar' ], 200 );
+
+		// Detect Elementor documents via their css that printed before the Admin Bar.
+		add_action( 'elementor/css-file/post/enqueue', [ $this, 'add_document_to_admin_bar' ] );
```

Comment:
Why don't you call `add_document_to_admin_bar` directly from where you enqueue the CSS?

Reply:
Because they are not related. frontend.php is the controller.

Reply:
I still think this is not the right way to deal with that, but I won't delay the release for that